geometry_msgs/PoseStamped pose
sensor_msgs/LaserScan scan